pagopa/io-app
# chore: [IOBP-73] Addition of payment survey banner

## Short description
added a banner that links to a survey on successful payment
<img width="250" alt="Screenshot 2023-06-21 at 17 33 46" src="https://github.com/pagopa/io-app/assets/60693085/d7a4c65b-5e13-438e-b92a-ff29c7822053">

## List of changes proposed in this pull request
- addition of required i18n keys
- usage of new webView to display said survey
- added mixpanel event to track started surveys
- added required pictogram

## How to test
wallet -> "paga un avviso" -> `io-dev-server` is your friend here, so fill with zeroes both fields -> get to the success screen and make sure everything follows design
